hack/release/RELEASING.md

```diff
@@ -205,6 +205,12 @@ When starting development on a new minor release, the first step is to create a
    git add calico/_includes/release-notes/<VERSION>-release-notes.md
    ```
 
+1. Update the Calico for Windows installation script.
+
+   ```
+   make -C calico install-calico-windows-script GIT_VERSION=<VERSION>
```

Comment:
Would be nice to not need a manual step for this - can't it be handled by our templating logic in the same way our manifests work?

Reply:
Yeah you're right, I should just revert the install-calico-windows.ps1 script back to a Liquid template.

Reply:
I've fixed this now. The install script was reverted to https://github.com/projectcalico/calico/blob/29c434e2b5330a021da7bb998cb40d3a044bc22d/scripts/install-calico-windows.ps1 with some fixes over top like https://github.com/projectcalico/calico/blob/552dae20400055f6023d4378e1e416821322b333/calico/scripts/install-calico-windows.ps1